spec/models/course/forum/topic_spec.rb

```diff
@@ -37,6 +37,18 @@
         end
       end
 
+      context 'when reinitializing a topic' do
+        subject do
+          test_topic = Course::Forum::Topic.new(posts_attributes: [title: nil])
+          test_topic.save
+          test_topic.posts.size
+        end
+
+        it 'does not create another post' do
+          is_expected.to eq(1)
```

Comment:
Then `expect(subject.posts.size).to eq(1)` here.  
Do you think it's better ? Using a count as subject is a bit weird.